### PR TITLE
Proposing Seymour Cray

### PR DESCRIPTION
**Pull request by cstivers78**

### CODE REVIEW

File: pkg/namesgenerator/names-generator.go [hunk 1]
```diff
@@ -119,6 +119,9 @@ var (
 		// Gerty Theresa Cori - American biochemist who became the third woman—and first American woman—to win a Nobel Prize in science, and the first woman to be awarded the Nobel Prize in Physiology or Medicine. Cori was born in Prague. https://en.wikipedia.org/wiki/Gerty_Cori
 		"cori",
 
+		// Seymour Roger Cray was an American electrical engineer and supercomputer architect who designed a series of computers that were the fastest in the world for decades. http://en.wikipedia.org/wiki/Seymour_Cray.
```
Comment (thaJeztah) [inline pkg/namesgenerator/names-generator.go]: Could you change the link to be `https://...` (for consistency)?

**Comment by thaJeztah** [comment]
Nice, thanks for contributing!

**Comment by icecrime** [comment]
@cstivers78 Good one thanks! But I agree with @thaJeztah, could you please change to `https://` and amend your commit (the same way you did earlier to fix the missing signature)?

**Comment by cstivers78** [comment]
I made the change. Now using https

**Comment by icecrime** [comment]
LGTM :+1:

**Comment by thaJeztah** [comment]
Thank you! lgtm (not a maintainer)

**Comment by crosbymichael** [comment]
LGTM

**Pull request merged**